### PR TITLE
Feat: Added Badges on Icons and Badges on Avatars

### PR DESCRIPTION
**Pull request by coldpigli**
Have added the badge component for Icons and Avatars.

badge-icon
badge-avatar

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    right: -10px;
+}
+
+.bottom-right{
+    bottom: 0;
+    right: 0.5rem;
+}
+
+.online{
+    background-color: var(--green);
```
Comment (rahulcs754) [inline css/components/badge.css:37]: name couild be green-color .

File: css/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.offline{
+    background-color: var(--gray);
```
Comment (rahulcs754) [inline css/components/badge.css:41]: same as above

File: css/components/badge.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.busy{
+    background-color: var(--cherry);
```
Comment (rahulcs754) [inline css/components/badge.css:45]: same as above

File: css/components/badge.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.away{
+    background-color: var(--orange);
```
Comment (rahulcs754) [inline css/components/badge.css:49]: same as above

**Review by rahulcs754 — COMMENTED** [review]
I liked you mostly used varaible name for size and color

**Review by raman1999 — COMMENTED** [review]
everything looks good to merge

File: css/components/badge.css (hunk 1 above)
Comment (coldpigli) [reply css/components/badge.css:37]: Thanks for the review. but naming it online is a conscious decision made to show state of user.

**Comment by coldpigli** [comment]
> everything looks good to merge

Thanks

**Pull request merged**